client/src/components/admin/ViewEditUsers/ViewEditUsers.jsx

```diff
@@ -96,6 +96,12 @@ class ViewEditUsers extends Component {
         }
     
 
+
```

Comment:
ViewEditUsers is already defined as a stateful component. Lines 99-104 need to be deleted.